language-support/ts/daml-react/templateStore.ts

```diff
@@ -66,14 +67,23 @@ export const payloadMatchesQuery = <T>(payload: T, query: Query<T>): boolean =>
 }
 
 
-// TODO(MH): We need to update the key lookups as well.
 export const addEvents = <T extends object, K>(store: Store<T, K>, events: Event<T, K>[]): Store<T, K> => {
```

Comment:
@robin-da Although this is already merged, can you please give it a proper review. This is a critical piece of code and you're the other person who knows what's going on here.

Reply:
:eyes: